extensions/azurecore/src/azurecore.d.ts

```diff
@@ -76,10 +80,16 @@ declare module 'azurecore' {
 		provideResources(): azureResource.IAzureResourceProvider[];
 
 		runGraphQuery<T extends azureResource.AzureGraphResource>(account: azdata.Account, subscriptions: azureResource.AzureResourceSubscription[], ignoreErrors: boolean, query: string): Promise<ResourceQueryResult<T>>;
+		makeHttpGetRequest(account: azdata.Account, subscription: azureResource.AzureResourceSubscription, ignoreErrors: boolean, url: string): Promise<HttpGetRequestResult>;
 	}
 
 	export type GetSubscriptionsResult = { subscriptions: azureResource.AzureResourceSubscription[], errors: Error[] };
 	export type GetResourceGroupsResult = { resourceGroups: azureResource.AzureResourceResourceGroup[], errors: Error[] };
+	export type GetSqlManagedInstancesResult = { resources: azureResource.AzureGraphResource[], errors: Error[] };
```

Comment:
Is there not a more specific type you can add for these? Some properties that only exist on each of the types?

Reply:
Should I manually create a type for it?

Reply:
Yeah - that'd be useful if these have extra properties on them that people might want to access. It'd probably be good to make a type even if it just extends AzureGraphResource without adding any new properties just to be more clear about what's being returned and making it easier to add properties later on.